init.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+eval $(printenv | sed -n "s/^\([^=]\+\)=\(.*\)$/export \1=\2/p" | sed 's/"/\\\"/g' | sed '/=/s//="/' | sed 's/$/"/' >> /etc/profile)
+
+echo "Intialising OpenRC..."
```

Comment:
As above, we should be able to start the server without this file

Reply:
We can, however this is a recommended approach when both server build and run is being executed.